prow/test/integration/setup-prow-components.sh

```diff
@@ -260,9 +260,9 @@ function deploy_prow() {
   # enough to only redeploy those components who configurations have changed as
   # a result of newly built images (from build_prow_images()).
   pushd "${SCRIPT_ROOT}/config/prow"
-  do_kubectl create configmap config --from-file=./config.yaml --dry-run=client -oyaml | do_kubectl apply -f - &
-  do_kubectl create configmap plugins --from-file=./plugins.yaml --dry-run=client -oyaml | do_kubectl apply -f - &
-  do_kubectl create configmap job-config --from-file=./jobs --dry-run=client -oyaml | do_kubectl apply -f - &
+  do_kubectl create configmap config --from-file=./config.yaml --dry-run=client -oyaml | do_kubectl apply -f -
+  do_kubectl create configmap plugins --from-file=./plugins.yaml --dry-run=client -oyaml | do_kubectl apply -f -
+  do_kubectl create configmap job-config --from-file=./jobs --dry-run=client -oyaml | do_kubectl apply -f -
```

Comment:
There are other places in this file where we use `do_kubectl` with `&` to run it in the background. We should probably fix all these consistently.

Reply:
the only other place where `&` is used is https://github.com/kubernetes/test-infra/blob/de7fa069413c0bc63eeab8ec4fd1eeca1a5aaec0/prow/test/integration/setup-prow-components.sh#L308, which is in a large for loop that consists of `kubectl wait` which already does these checks. Wait ahead of `kubectl wait` loses the point of async, and wait afterwards doesn't have any effect, so will not handle these